docs/load-test-how-to-use.md

```diff
@@ -0,0 +1,115 @@
+
+## Steps to load test the feature
+
+### Brief summary of tasks
+
+The steps to load test a feature include:
+ - Writing new load testing actions to mattermost-load-test-ng.
+ - Testing the changes locally.
+ - Testing the changes in terraform: its purpose is to load-test with a larger dataset.
+ - Analyse load-test results.
+ - Getting the changes merged to the load-test repository.
+
+### Detailed summary of tasks
+
+#### Writing new load testing code
+
+ - Go through [coverage.md](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/coverage.md#implementation-overview), and make a list of changes needed to load-test the new feature.
+ - Optionally check out [this video walkthrough](https://drive.google.com/file/d/1l462zMdANwCRXUtj7nnHv2CX_6BiINHl/view) by @streamer45 
+ - Make the necessary changes in `loadtest/`.
+
+#### Testing changes locally
+
+ - Go through [local_loadtest.md](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/local_loadtest.md).
+ - Some additional information on the above document
+    - Updates to `config.json`
+        - Make sure to change `ConnectionConfiguration` section in `config.json` according to the local deployment of mattermost-server.
+        - For `InstanceConfiguration`, see [this doc](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/loadtest_config.md#instanceconfiguration). This configuration setting is used by the `init` command to initially populate the mattermost database.
+    - Increase the frequency of the new action, so it's easier to debug while running locally. [sample](https://github.com/mattermost/mattermost-load-test-ng/blob/8faa4dfb485dace3bd65908c0d3d98979b7dfd17/loadtest/control/simulcontroller/controller.go#L227)
+    - If you see errors, there's a [troubleshooting guide](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/faq.md#troubleshooting) you can reference to resolve the issues.
+    - Check `ltagent.log` located in the `mattermost-load-test-ng` directory, and the server logs for details on errors, if any.
+    - We highly recommend becoming familiar with additional sections of documentation, including [using load-test agent API server](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/local_loadtest.md#running-a-load-test-through-the-load-test-agent-api-server) and [using load-test coordinator](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/local_loadtest.md#running-a-load-test-through-the-coordinator), since the Terraform deployment uses the latter method to execute the load-tests, and it'll be easier to debug the setup if the you understand these underlying concepts and principles.
```

Comment:
```suggestion
    - We highly recommend becoming familiar with additional sections of documentation, including [using load-test agent API server](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/local_loadtest.md#running-a-load-test-through-the-load-test-agent-api-server) and [using load-test coordinator](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/local_loadtest.md#running-a-load-test-through-the-coordinator), since the Terraform deployment uses the latter method to execute the load-tests, and it'll be easier to debug the setup if you understand these underlying concepts and principles.
```